omnibus-arbitrum-sepolia-arbthetix.toml

```diff
@@ -0,0 +1,118 @@
+    #"tomls/omnibus-arbitrum-sepolia-arbthetix/collaterals/dai.toml", // dai collateral is already included with the spot market deployment
+    "tomls/collaterals/weth.toml",
+    "tomls/collaterals/arb.toml",
+    # BTC
+    "tomls/markets/perps/btc.toml",
+    "tomls/oracles/pyth-btc.toml",
+    "tomls/omnibus-arbitrum-sepolia-arbthetix/perps/btc.toml",
+    # ETH
+    "tomls/markets/perps/eth.toml",
+    "tomls/oracles/pyth-eth.toml",
+    "tomls/omnibus-arbitrum-sepolia-arbthetix/perps/eth.toml",
+]
+
+[setting.commitmentPriceDelay]
+defaultValue = "2"
+
+[setting.bigCapSettlementDelay]
+defaultValue = "2"
+
+[setting.snx_package]
+defaultValue = "synthetix:3.3.15"
+
+[setting.perps_keeper_cost_package]
+defaultValue = "arbitrum-gas-price-oracle:3.3.16"
+
+[setting.spot_market_package]
+defaultValue = "synthetix-spot-market:3.3.15"
+
+[setting.perps_market_package]
+defaultValue = "synthetix-perps-market:3.3.17"
+
+[setting.buyback_snx_package]
+defaultValue = "buyback-snx:3.3.14" # Do not update! This package isn't upgradeable. A version change will deploy a new contract.
+
+[setting.salt]
+defaultValue = "arbthetix"
+
+[setting.target_preset]
+defaultValue = "arbthetix" # Preset assigned to provisioned packages
+
+[setting.pyth_price_verification_address]
+# https://docs.pyth.network/documentation/pythnet-price-feeds/evm
+defaultValue = "0x4374e5a8b9C22271E9EB878A2AA31DE97DF15DAF"
+
+[provision.pyth_erc7412_wrapper]
+source = "pyth-erc7412-wrapper:3.3.14"
+targetPreset = "<%= settings.target_preset %>"
+options.salt = "<%= settings.salt %>"
+options.pythAddress = "<%= settings.pyth_price_verification_address %>"
+
+[setting.owner]
+defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
+
+[setting.deployer]
+defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
+
+[setting.minimum_liquidity_ratio]
+defaultValue = "<%= parseEther('1.25') %>" # 125% min market collateralization
+
+[setting.account_timeout_withdraw]
+defaultValue = "86400" # 1 day
+
+[setting.sc_pool_id]
+defaultValue = "1"
+
+[setting.pool_owner]
+defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
+
+[setting.arb_aggregator_address]
+defaultValue = "0xD1092a65338d049DB68D7Be6bD89d17a0929945e"
+
+[setting.eth_aggregator_address]
+defaultValue = "0xd30e2101a97dcbAeBCBC04F14C3f624E67A35165"
+
+[setting.weth_address]
+defaultValue = "0x980B62Da83eFf3D4576C647993b0c1D7faf17c73"
+
+[setting.arb_address]
+defaultValue = "0xc275B23C035a9d4EC8867b47f55427E0bDCe14cB"
+
+[invoke.setScPoolConfig]
+target = ["system.CoreProxy"]
+fromCall.func = "getPoolOwner"
+fromCall.args = ["<%= settings.sc_pool_id %>"]
+func = "setPoolConfiguration"
+args = [
+    "<%= settings.sc_pool_id %>",
+    [
+        { marketId = "<%= imports.perpsFactory.extras.superMarketId %>", weightD18 = 1, maxDebtShareValueD18 = "<%= parseEther('1') %>" },
+    ],
+]
+
+[invoke.permitAllPerpsSystem_upd_1]
```

Comment:
Gets executed in release simulation

<img width="1120" alt="image" src="https://github.com/Synthetixio/synthetix-deployments/assets/28145325/f4868415-d4ef-4fed-83c2-8fd785bc34c3">